### PR TITLE
Avatar and badge components

### PR DESCRIPTION
**Pull request by shagunbagrecha**
Avatar 
- Image Avatar
- Letter Avatar
- Text Avatar

Badge
- Badge on Avatars
- Badge on Icons

Preview-link : https://deploy-preview-3--vivid-ui.netlify.app/

### CODE REVIEW

File: components/Avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+.avatar-xs {
+	width: 4rem;
+	height: 4rem;
+}
+
+.avatar-xxs{
+	width: 3rem;
+	height: 3rem;
+}
```
Comment (gopalbbharadva) [inline components/Avatar/avatar.css]: Suggetion: This avatar maybe not used, so remote it, and keep avatar sizes consistent, like 11,9,7,5,3

File: components/Badge/badge.css [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+.badge-status-ill{
+	background-color: var(--clr-red);
+}
+
+.badge-icon{
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	color: white;
```
Comment (gopalbbharadva) [inline components/Badge/badge.css]: Suggetion: You should use a variable for static color as well, bcuz it may use multiple places in large file.

**Review by gopalbbharadva — APPROVED** [review]
Clean components!!

**Review by MarkVed17 — APPROVED** [review]
Just the changes @gopalbbharadva suggested and you’ll be good to proceed.

**Comment by preethasri** [comment]
UI looks good

**Pull request merged**